### PR TITLE
Add Thread Dump Analyser plugin

### PR DESCRIPTION
**Pull request by vasugr**
This is a thread dump analyser plugin, which takes a thread dump text file as an input and produces analytic data. I have made similar plugin for VScode as well , you can find it [here](https://marketplace.visualstudio.com/items?itemName=Vasu.tda)

### CODE REVIEW

File: repository/t.json [hunk 1]
```diff
@@ -2233,6 +2233,20 @@
 				}
 			]
 		},
+		{
+			"name": "Thread Dump Analyser",
+			"author": "Vasu",
+			"details": "https://github.com/vasugr/Thread-Dump-Analyser",
+			"readme": "https://raw.githubusercontent.com/vasugr/Thread-Dump-Analyser/main/README.md",
+			"issues": "https://github.com/vasugr/Thread-Dump-Analyser/issues",
```
Comment (braver) [inline repository/t.json:2241]: Your README and issues are in the default location, so you don't need to link to them here (ie. please remove these lines)

Comment (vasugr) [reply repository/t.json]: Thanks. I have made the changes.

**Comment by rchl** [comment]
Just out of curiosity, is the thread dump format universal? If not then would it make sense to specify what is supported?

**Comment by braver** [comment]
We don't recommend shipping with keybindings, especially for packages that don't completely need them. They tend to conflict with those of other packages, or even existing defaults. Just make sure your commands are in the command palette, that's enough for most users and usually as good as or even better than keybindings (in any case a lot easier to discover). Then maybe document how to make the bindings in the README, so users can choose their own bindings.

You have some .DS_Store files in your repo, usually you want to have those covered by `.gitignore`.

You have some PNG's in the docs. You can exclude such non-functional files from the package (saving on storage, bandwidth and CO2) using a .gitattributes file.

Python files in subdirectories are not hot-reloaded, making it a bit harder during development. It doesn't look like you really need a "models" directory right now?

**Comment by braver** [comment]
Please respond to the feedback to continue this PR.

**Comment by vasugr** [comment]
I'll make the aforementioned changes by month end.

**Comment by braver** [comment]
September is gone. Any news?

**Comment by vasugr** [comment]
Thanks for the reminder. I will complete it by 2 weeks. Thank you

**Comment by braver** [comment]
Please let us know when you're ready, we can re-open the PR then.

**Pull request closed without merging**